diff --git a/pkg/cmd/openshift-tests/run/options.go b/pkg/cmd/openshift-tests/run/options.go
index c5f9715a47..a025da656c 100644
--- a/pkg/cmd/openshift-tests/run/options.go
+++ b/pkg/cmd/openshift-tests/run/options.go
@@ -97,6 +97,7 @@ func (o *RunSuiteOptions) Run(ctx context.Context) error {
 		ClusterStabilityDuringTest: monitortestframework.ClusterStabilityDuringTest(stabilitySetting),
 		ExactMonitorTests:          o.GinkgoRunSuiteOptions.ExactMonitorTests,
 		DisableMonitorTests:        o.GinkgoRunSuiteOptions.DisableMonitorTests,
+		NoMonitorTests:             o.GinkgoRunSuiteOptions.NoMonitorTests,
 	}
 
 	o.GinkgoRunSuiteOptions.CommandEnv = o.TestCommandEnvironment()
diff --git a/pkg/defaultmonitortests/types.go b/pkg/defaultmonitortests/types.go
index 84f95b5deb..d2d74eded3 100644
--- a/pkg/defaultmonitortests/types.go
+++ b/pkg/defaultmonitortests/types.go
@@ -93,6 +93,11 @@ func NewMonitorTestsFor(info monitortestframework.MonitorTestInitializationInfo)
 	// get tests and apply any filtering defined in info
 	var startingRegistry monitortestframework.MonitorTestRegistry
 
+	if info.NoMonitorTests {
+		// return an empty monitor test registry
+		return monitortestframework.NewMonitorTestRegistry(), nil
+	}
+
 	switch info.ClusterStabilityDuringTest {
 	case monitortestframework.Stable:
 		startingRegistry = newDefaultMonitorTests(info)
diff --git a/pkg/monitortestframework/types.go b/pkg/monitortestframework/types.go
index 87e83c8a27..39fa0a3614 100644
--- a/pkg/monitortestframework/types.go
+++ b/pkg/monitortestframework/types.go
@@ -34,6 +34,7 @@ type MonitorTestInitializationInfo struct {
 
 	// DisableMonitorTests will remove any monitor tests contained in the provided list
 	DisableMonitorTests []string
+	NoMonitorTests      bool
 }
 
 type OpenshiftTestImageGetterFunc func(ctx context.Context, adminRESTConfig *rest.Config) (imagePullSpec string, notSupportedReason string, err error)
diff --git a/pkg/test/ginkgo/cmd_runsuite.go b/pkg/test/ginkgo/cmd_runsuite.go
index 11efd78d57..fa18b9c6a9 100644
--- a/pkg/test/ginkgo/cmd_runsuite.go
+++ b/pkg/test/ginkgo/cmd_runsuite.go
@@ -93,6 +93,7 @@ type GinkgoRunSuiteOptions struct {
 
 	ExactMonitorTests   []string
 	DisableMonitorTests []string
+	NoMonitorTests      bool
 	Extension           *extension.Extension
 
 	// RetryStrategy controls retry behavior and final outcome decisions
@@ -131,6 +132,7 @@ func (o *GinkgoRunSuiteOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.StringSliceVar(&o.ExactMonitorTests, "monitor", o.ExactMonitorTests,
 		fmt.Sprintf("list of exactly which monitors to enable. All others will be disabled.  Current monitors are: [%s]", strings.Join(monitorNames, ", ")))
 	flags.StringSliceVar(&o.DisableMonitorTests, "disable-monitor", o.DisableMonitorTests, "list of monitors to disable.  Defaults for others will be honored.")
+	flags.BoolVar(&o.NoMonitorTests, "no-monitor", o.NoMonitorTests, "Do not run any monitors.")
 
 	flags.IntVar(&o.ShardID, "shard-id", o.ShardID, "When tests are sharded across instances, which instance we are")
 	flags.IntVar(&o.ShardCount, "shard-count", o.ShardCount, "Number of shards used to run tests across multiple instances")
diff --git a/pkg/testsuites/standard_suites.go b/pkg/testsuites/standard_suites.go
index f7d51800f1..ad2fec2e64 100644
--- a/pkg/testsuites/standard_suites.go
+++ b/pkg/testsuites/standard_suites.go
@@ -151,7 +151,7 @@ var staticSuites = []ginkgo.TestSuite{
 	{
 		Name: "openshift/disruptive",
 		Description: templates.LongDesc(`
-		The disruptive test suite.  Disruptive tests interrupt the cluster function such as by stopping/restarting the control plane or 
+		The disruptive test suite.  Disruptive tests interrupt the cluster function such as by stopping/restarting the control plane or
 		changing the global cluster configuration in a way that can affect other tests.
 		`),
 		Qualifiers: []string{
@@ -317,9 +317,9 @@ var staticSuites = []ginkgo.TestSuite{
 		The conformance testing suite for certified third-party CNI plugins.
 		`),
 		Qualifiers: []string{
-			`name.contains("[Suite:k8s]") && name.contains("[sig-network]") && 
-				(name.contains("[Conformance]") || 
-					(name.contains("NetworkPolicy") && !name.contains("named port")) || 
+			`name.contains("[Suite:k8s]") && name.contains("[sig-network]") &&
+				(name.contains("[Conformance]") ||
+					(name.contains("NetworkPolicy") && !name.contains("named port")) ||
 					name.contains("[Feature:IPv6DualStack]"))`,
 		},
 	},
@@ -478,6 +478,15 @@ var staticSuites = []ginkgo.TestSuite{
 		TestTimeout:                40 * time.Minute,
 		ClusterStabilityDuringTest: ginkgo.Disruptive,
 	},
+	{
+		Name: "liouk-devel",
+		Description: templates.LongDesc(`
+		Run only the tests that have "[liouk-devel]" in their name.
+		`),
+		Qualifiers: []string{
+			`name.contains("[liouk-devel]")`,
+		},
+	},
 }
 
 func withExcludedTestsFilter(baseExpr string) string {
